spyder/plugins/application/container.py

```diff
@@ -326,6 +324,8 @@ def _check_updates_ready(self):
                     self.application_update_status.start_installation(
                         latest_release=latest_release)
 
+                    return
```

Comment:
I don't think that we can have a return statement `_check_updates_ready`, since code on lines 419-425 must always be executed.

Reply:
You're right, sorry for not understanding your initial message about it correctly.